gatorgrade/generate/generate.py

```diff
@@ -1,21 +1,82 @@
-"""Generate a GatorGrade configuration file.
+"""Generate a YAML file with default messages and specific paths."""
+import os
+from typing import List
 
-Gatorgrade file will include paths to whitelisted files
-and default GatorGrader checks.
-"""
+# Define colors for terminal output
+OKGREEN = "\033[92m"
+WARNING = "\033[93m"
+FAIL = "\033[91m"
 
-# Import the necessary libraries
 
-
-def create_targeted_paths_list():
+def create_targeted_paths_list(
+    key_word_list: List[str], relative_run_path: str = "."
```

Comment:
I love that you've specified the parameter types this is amazing! 😍

Reply:
Thanks for your appreciation!